hikari/impl/shard.py

```diff
@@ -235,6 +236,9 @@ def __init__(
     ) -> None:
         self._activity: undefined.UndefinedNoneOr[presences.Activity] = initial_activity
         self._backoff = rate_limits.ExponentialBackOff(base=1.85, maximum=600, initial_increment=2)
+        # We limit member chunk requests to a stricter ratelimit of 60 requests per minute as to ensure that chunk
+        # requests don't single-handedly exhaust the request limit when being triggered on mass.
+        self._chunk_request_ratelimiter = rate_limits.WindowedBurstRateLimiter(f"guild chunking on {shard_id}", 60, 60)
```

Comment:
This name should probably have ratelimit in it

Reply:
should i also change the name for self._ratelimiter then?